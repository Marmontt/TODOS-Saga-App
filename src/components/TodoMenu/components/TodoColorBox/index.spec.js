import React from "react";
import { shallow } from "enzyme";

import { TodoColorBox } from "./index";

const props = {
  chosenColor: "red",
  color: "yellow",
  setColor: jest.fn()
};

describe("TodoColorBox", () => {
  it("should render properly", () => {
    const wrapper = shallow(<TodoColorBox {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
  it("should change color", () => {
    const wrapper = shallow(<TodoColorBox {...props} />);
    wrapper.simulate("click");
    expect(props.setColor).toBeCalledWith("yellow");
  });
});
