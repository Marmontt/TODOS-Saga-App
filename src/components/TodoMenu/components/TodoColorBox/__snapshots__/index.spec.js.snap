// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoColorBox should render properly 1`] = `
<div
  className="box"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "yellow",
    }
  }
/>
`;
